README.md

```diff
@@ -1,2 +1,127 @@
+  uses: AbsaOSS/living-doc-generator@v0.1.0
+  env:
+    GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}  
+  with:
+    TODO
+```
+
+### Configure the Action
+Configure the action by customizing the following parameters based on your needs:
+
+- **GITHUB_TOKEN** (required): Your GitHub token for authentication. Store it as a secret and reference it in the workflow file as secrets.GITHUB_TOKEN.
+- **TODO** (required): TODO
+
+## Setup
+If you need to build the action locally, follow these steps:
+
+#### Prepare the Environment
+```
+node --version
+python3 --version
+```
+
+#### Install Node.js Dependencies
+```
+npm install
+```
+
+#### Compile or Prepare the JavaScript Files
+```
+npm run build
+```
+
+#### Set Up Python Environment
+```
+python3 -m venv venv
+source venv/bin/activate
+pip install -r requirements.txt
+```
+
+### Run unit test
+#### Launch unit tests
+```
+pytest
+```
+
+#### To run specific tests or get verbose output:
+```
+pytest -v  # Verbose mode
+pytest path/to/test_file.py  # Run specific test file
+```
+
+#### To check Test Coverage:
+```
+pytest --cov=../scripts
+```
+
+#### After running the tests
+```
+deactivate
+```
+
+#### Commit Changes
+After testing and ensuring that everything is functioning as expected, prepare your files for deployment:
+
+```
+git add action.yml dist/index.js  # Adjust paths as needed
+git commit -m "Prepare GitHub Action for deployment"
+git push
+```
+
+### Deployment
+This project uses GitHub Actions for deployment draft creation. The deployment process is semi-automated by a workflow defined in `.github/workflows/release_draft.yml`.
+
+- **Trigger the workflow**: The `release_draft.yml` workflow is triggered on workflow_dispatch.
+- **Create a new draft release**: The workflow creates a new draft release in the repository.
+- **Finalize the release draft**: Edit the draft release to add a title, description, and any other necessary details related to GitHub Action.
+- **Publish the release**: Once the draft is ready, publish the release to make it available to the public.
+
+
+## Features
+
+TODO - project states scan for Features
+
+
+### Contribution Guidelines
+
+We welcome contributions to the Living Documentation Generator! Whether you're fixing bugs, improving documentation, or proposing new features, your help is appreciated.
+
+#### How to Contribute
+- **Submit Pull Requests**: Feel free to fork the repository, make changes, and submit a pull request. Please ensure your code adheres to the existing style and all tests pass.
+- **Report Issues**: If you encounter any bugs or issues, please report them via the repository's [Issues page](https://github.com/AbsaOSS/living-doc-generator/issues).
+- **Suggest Enhancements**: Have ideas on how to make this action better? Open an issue to suggest enhancements.
+
+Before contributing, please review our [contribution guidelines](https://github.com/AbsaOSS/living-doc-generator/blob/master/CONTRIBUTING.md) for more detailed information.
```

Comment:
```suggestion
Before contributing, please review our [contribution guidelines](https://github.com/AbsaOSS/living-doc-generator/blob/master/CONTRIBUTING.md) for more detailed information.
```

Reply:
Addressed in commit - bcb84a552552cb710f87486bbee433af62b38f6d.